example_scripts/run_simulation_OLD.py

```diff
@@ -0,0 +1,182 @@
+import time
```

Comment:
this file is called run_simulation_OLD.py. Should we delete it?

Reply:
Yep will remove it